server/initializers/constants.ts

```diff
@@ -116,8 +116,9 @@ const CONFIG = {
     PASSWORD: config.get<string>('database.password')
   },
   REDIS: {
-    HOSTNAME: config.get<string>('redis.hostname'),
-    PORT: config.get<number>('redis.port'),
+    get HOSTNAME () { try { return config.get<string>('redis.hostname') } catch { return null } },
```

Comment:
These can probably be multiline

Reply:
But then we lose the type constraint

Reply:
Ah right, forget about what I just said :blush: